src/collapse/docs/readme.md

```diff
@@ -7,3 +7,27 @@
   <i class="glyphicon glyphicon-eye-open"></i>
   _(Default: `false`)_ -
   Whether the element should be collapsed or not.
+
```

Comment:
Check that, @wesleycho, what do you think about that last bit?  Function vs. expression?